src/main/java/digital/slovensko/autogram/server/SignEndpoint.java

```diff
@@ -33,7 +36,7 @@ public void handle(HttpExchange exchange) throws IOException {
         try {
             var body = EndpointUtils.loadFromJsonExchange(exchange, SignRequestBody.class);
             var job = new SigningJob(body.getDocument(), body.getParameters(), new ServerResponder(exchange));
-            autogram.sign(job);
+            ui.sign(autogram, job);
```

Comment:
preco to nie je cez `autogram.sign` ked autogram uz ma referenciu na `ui`?

Reply:
Volanie ```ui.sign``` je kvoli tomu ze pri GUI sa v dvoch threadoch pusti podpisovanie a pdfa compliance check, pri CLI sa najprv prebehnu vsetky subory a zbehne pdfa compliance check a az potom sa ide na podpisovanie. Mozme iba overloadnut autogram.sign metodu, otazka je ci to chceme mat rozdelene na urovni autogramu alebo na urovni ui.

Reply:
toto a všetky súvisiace pripomienky vyššie už asi nie sú relevantné, že? V kóde pozerám, že ui.sign už neexistuje.

Reply:
@Bombino1024 @pomali

Reply:
ano, to je uz vyriesene